pkg/cli/build.go

```diff
@@ -113,7 +113,7 @@ func GenBuildOptions(c *cobra.Command, inputArgs []string, iopts BuildOptions) (
 	if c.Flag("build-context").Changed {
 		for _, contextString := range iopts.BuildContext {
 			av := strings.SplitN(contextString, "=", 2)
-			if len(av) > 1 {
+			if len(av) > 1 && av[0] != "" && av[1] != "" {
```

Comment:
This is basically so we skip over `-v :` or something? This could use a comment, maybe even we have a `const DefaultBuildContextKey = ""` and then reference it here?

Reply:
Pushing the check on `av[1]` down into `GetAdditionalBuildContext()` makes it a little easier to follow, added a comment here and put more checks in there.